Cargo.toml

```diff
@@ -1,16 +1,20 @@
 [package]
 name = "m8-files"
 description = "Parse the Dirtywave M8's file formats"
-version = "0.2.0"
+version = "0.3.0"
 edition = "2021"
 repository = "https://github.com/AlexCharlton/m8-files"
 documentation = "https://docs.rs/m8-files"
 homepage = "https://github.com/AlexCharlton/m8-files"
 license = "MIT"
 keywords = ["music"]
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
-
 [dependencies]
 arr_macro = "0.2.1"
+array-concat = "0.5.3"
 byteorder = "1.4.3"
+num_enum = "0.7.3"
+
+[features]
+# Used to squash compilation warnings stemming from array-concat
+"const_panic"=[]
```

Comment:
How so? I don't see it used...

Reply:
I use some `concat_arrays!` macros to compose some array for FX mapping, and it's not happy if the feature is not defined.

Reply:
Ah, I see. Annoyingly, this is a clear bug of that crate: https://github.com/inspier/array-concat/pull/4

Can you add a note to that comment saying:
```
This feature can be removed when this bug is fixed: https://github.com/inspier/array-concat/pull/4
```